bin/train.py

```diff
@@ -56,7 +55,6 @@
     type=str,
     default="titan",
     help="Dataset to use",
-    choices=dataset_registry.keys(),
```

Comment:
can we keep the choices of datasets by retrieving the data accessor keys ?

Reply:
Well, if I have a titan dataset with name "titan_lbc", I want the code to accept it, and recognize that it should use the "titan" accessor. (at least that's the idea, maybe not the best one). Otherwise this is quite rigid in defining names.